.gitmodules

```diff
@@ -13,3 +13,6 @@
 [submodule "packages/contracts-bedrock/lib/forge-std"]
 	path = packages/contracts-bedrock/lib/forge-std
 	url = https://github.com/foundry-rs/forge-std
+[submodule "packages/contracts-bedrock/lib/safe-contracts"]
```

Comment:
We should be sure that the correct tag is used, i dont think we want `master`

Reply:
https://github.com/safe-global/safe-contracts/tree/v1.4.0

`master` contains not prod ready code

Reply:
great call out